yb-voyager/cmd/root.go

```diff
@@ -207,6 +214,9 @@ func registerCommonGlobalFlags(cmd *cobra.Command) {
 
 	registerExportDirFlag(cmd)
 
+	cmd.PersistentFlags().StringVarP(&logLevel, "log-level", "l", "info",
```

Comment:
Add validation for this flag and we can check for case-insensitive checks for the accepted values

Reply:
Just thinking: Should we keep it hidden for now until we have a proper different kinds of logging in all the commands?

Reply:
>Add validation for this flag and we can check for case-insensitive checks for the accepted values

the logrus library already throws an error by doing a case insensitive check: `Failed to initialize logging: invalid log level erroe: not a valid logrus Level: "erroe"` , but yeah, I think we can do our own validation, and also probably convert it to lower-case, so that we don't have to lower-case it everywhere we're comparing..

Reply:
>Just thinking: Should we keep it hidden for now until we have a proper different kinds of logging in all the commands?

We can incrementally add debug logs for various scenarios, but don't think we should restrict users from being able to specify it. We do have a tiny amount of debug logs! :D

Reply:
I was thinking in terms of how the user will take this as earlier we used to have this `--verbose` flag right but we were not printing anything specific as such with verbose mode and that created confusion with a user that it doesn't or something like that.